src/main/java/onlinebookstore/model/Book.java

```diff
@@ -0,0 +1,29 @@
+package onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Table(name = "books")
+@Data
```

Comment:
be careful with this annotation, since it includes `@EqualsAndHashCode`

see https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/#dont-use-equalsandhashcode